<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Minerva Zigbee Application Demo v1.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/logo-no-text.png</normaloff>images/logo-no-text.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="labelConnect">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Waiting connect</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTreeView" name="viewItem">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ContiguousSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTableView" name="viewProperty">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QListWidget" name="listWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="actionConnect"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConnection"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
